component-samples/demo/aio/aio.env

```diff
@@ -0,0 +1,54 @@
+aio_port=8080
+aio_database_connection_url=jdbc:h2:tcp://localhost:8081/./aio
+aio_database_username=sa
+aio_database_password=
+aio_database_port=8081
+aio_database_driver=org.h2.Driver
+aio_database_allow_others=true
```

Comment:
The default behavior should be false.

We should also print a warning message if this is enabled, stating that "This property should NOT be set to true in production deployments."